src/main/java/bio/terra/service/snapshot/SnapshotDao.java

```diff
@@ -29,7 +29,7 @@
 import java.util.UUID;
 
 @Repository
-public class SnapshotDao {
+public class  SnapshotDao {
```

Comment:
nit: extra space

Reply:
Weird, I didn't think I changed that file... but fixed